File: src/PHP/StructuralPatterns/Composite/WindowsFormsApplication/Composites/GroupBox.php
<?php

namespace StructuralPatterns\Composite\WindowsFormsApplication\Composites;

class GroupBox extends BaseControlComposite
{
    public function __construct($name, $width, $height, $foreColor, $backColor)
    {
        parent::__construct($name, $width, $height, $foreColor, $backColor);
    }
}

File: src/C#/StructuralPatterns/Composite/WindowsFormsApplication/Composites/GroupBox.cs
using System.Drawing;
using WindowsFormsApplication.Composites.Abstraction;

namespace WindowsFormsApplication.Composites
{
    public class GroupBox : BaseControlComposite
    {
        public GroupBox(string name, int width, int height, Color foreColor, Color backColor) 
            : base(name, width, height, foreColor, backColor)
        {

        }
    }
}
